src/main/java/org/opentestsystem/ap/ims/rest/v1/ItemApi.java

```diff
@@ -88,6 +92,31 @@ public Item getItem(@PathVariable final String itemId) {
      * @param itemId The unique item identifier.
      * @return The changes associated with an item.
      */
+    @GetMapping("/{itemId}/history/csv")
+    public void getItemHistoryCsv(@PathVariable final String itemId, final HttpServletResponse response) {
+        log.debug("find item csv {}", itemId);
+
+        final String csvString = itemService.findItemHistoryCsvString(itemId);
+        InputStream csvStream = null;
+        try {
+            csvStream = IOUtils.toInputStream(csvString, "UTF-8");
```

Comment:
is making the steam encoding UTF-8 equivalent to setting the request.setEncoding("utf-8") ?